libs/sandbox-py/langchain_sandbox/pyodide.py

```diff
@@ -79,17 +92,18 @@ class PyodideSandbox:
     - Streaming stdout/stderr capture
     """
 
-    def __init__(  # noqa: PLR0913
+    def __init__(
         self,
-        sessions_dir: str,
         *,
+        stateful: bool = False,
         allow_env: list[str] | bool = False,
         allow_read: list[str] | bool = False,
         allow_write: list[str] | bool = False,
         allow_net: list[str] | bool = False,
         allow_run: list[str] | bool = False,
         allow_ffi: list[str] | bool = False,
         node_modules_dir: str = "auto",
+        pyodide_package: str = PKG_NAME,
```

Comment:
I was hoping that we don't make this controllable by the user since the python packages are should usually be 1 to 1 w/ the typescript package

Reply:
let me see if i can patch this somehow for unittesting